packages/client/mutations/EndDraggingReflectionMutation.ts

```diff
@@ -48,6 +48,7 @@ graphql`
       id
       dragUserId
       dragUserName
+      isSpotlight
```

Comment:
Add this so `remoteDrag?.isSpotlight` is no longer true when the Spotlight closes